### DISCUSSION TITLE
Product page, attribute section, filter enum/list attribute

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#576] [Q&A]
When editing product attribute with type list/enum, there is no search
Is it possible in pim to enable search for attribute values in attribute with type list like in second picture 
![image](https://user-images.githubusercontent.com/30953478/208827663-6da462bd-900a-4946-a39f-45b931b19007.png)

![image](https://user-images.githubusercontent.com/30953478/208827915-1dec832b-4f93-4c7a-bae7-0b2531ed440d.png)

**Comment by atrolex** [comment]
If you start typing you will be scrolled to appropriate position (which starts with what you are typing), a visible field for a filter is planed, will be implemented someday in 2023.
